src/app/components/feed/FeedClusters.module.css

```diff
@@ -12,6 +12,14 @@
 
 .feedClusterCard {
   margin-bottom: 16px;
+
```

Comment:
I think that if you do the above suggestion, we won't need the hover styling here because it'll be already active on `Card`.

Reply:
Confirmed, done!